String/Palindrome.js

```diff
@@ -0,0 +1,21 @@
+// Wikipedia: https://en.wikipedia.org/wiki/Palindrome
+
+function isPalindrome (string) {
```

Comment:
this algo already exists #213

Reply:
Sorry, I didn't see it. Should I close the PR?